Modules/_sqlite/cache.c

```diff
@@ -291,7 +297,8 @@ static PyType_Spec cache_spec = {
 };
 PyTypeObject *pysqlite_CacheType = NULL;
 
-extern int pysqlite_cache_setup_types(PyObject *mod)
+int
+pysqlite_cache_setup_types(PyObject *mod)
```

Comment:
It would be great if you could submit a separate PR for removing redundant `extern`s.

Reply:
I can remove the `extern` clean-up and submit a separate PR for those using the same issue number.